components/clp-package-utils/clp_package_utils/scripts/archive_manager.py

```
@@ -171,9 +172,11 @@ def main(argv: List[str]) -> int:
     try:
         config_file_path: Path = Path(parsed_args.config)
         clp_config: CLPConfig = load_config_file(
-            config_file_path, default_config_file_path, clp_home
+            resolve_host_path(config_file_path),
+            resolve_host_path(default_config_file_path),
+            clp_home,
         )
-        clp_config.validate_logs_dir()
+        clp_config.validate_logs_dir(True)
```

Ruff (0.14.2)

Boolean positional value in function call

(FBT003)